doc/man/bitcoind.1

```diff
@@ -530,7 +530,7 @@ option can be specified multiple times
 .HP
 \fB\-rpcauth=\fR<userpw>
 .IP
-Username and hashed password for JSON\-RPC connections. The field
+Username and HMAC-SHA-256 hashed password for JSON\-RPC connections. The field
```

Comment:
No need to modify these files., they are autogenerated.

Reply:
Oh? If they're autogenerated then why are they version controlled?

Reply:
https://github.com/bitcoin/bitcoin/pull/8608#issuecomment-242899755